meta-balena-common/recipes-containers/balena/balena/balena.service

```diff
@@ -36,12 +36,16 @@ OOMScoreAdjust=-900
 MountFlags=slave
 LimitNOFILE=1048576
 LimitNPROC=1048576
-LimitCORE=infinity
+# Don't generate core dumps. This way, user apps that are crashlooping will not
+# fill up all storage space.
+LimitCORE=0
```

Comment:
@lmbarros this is definitely right for production mode images, however, I think development mode images should keep the setting as it was so container dumps can be analyzed.

I am thinking that having something like:
```
LimitCORE=${BALENA_LIMIT_CORE:-0}
```
And then define that in the `/var/volatile/development-features` file that is only created in development mode might work.

Could you maybe play with that idea see if there is something there?

Reply:
Maybe, instead of running healthdog/balenad directly, we can run a script that calls `ulimit -c` with an argument than depends in `BALENA_LIMIT_CORE`, then `exec`s healthdog/balenad. A bit convoluted to do that outside of Systemd, but may work. I'll try this.